### PR TITLE
Updated instructions for importing exam grades and fixed markup typo

### PR DESCRIPTION
**Pull request by briangrossman**
#### What's this PR do?

Update the grades readme
   - updated instructions for importing exam grades
   - fixed a markdown typo 

#### How should this be manually tested?

View the readme

### CODE REVIEW

File: grades/README.md [hunk 1]
```diff
@@ -127,67 +127,77 @@ and type (being ABSOLUTELY sure of what you are doing):
...
 
-### Load adjusted grades
+Note that users will have usernames from the platform on which they took the exams (e.g. MITx Online). If necessary, use the social_auths to convert the usernames of the exam platform to Micromasters usernames. 
+
+Steps:
+- Copy the spreadsheet to your computer
+- If necessary replace usernames with Micromasters usernames (see note above)
+- Copy the spreadsheet in your pasteboard: 
+   - `cat <.csv file> | pbcopy`
+   - Connect to the Hiroku instance for MM
```
Comment (annagav) [inline grades/README.md:161]: typo: Heroku

File: grades/README.md [hunk 2]
```diff
@@ -127,67 +127,77 @@ and type (being ABSOLUTELY sure of what you are doing):
...
 
-Both steps (1. validating the number of learners who have earned different numbers of course certificates and 2. 
-receiving the email confirmation from the course team to release the grades) need to occur before importing the 
-grades into the micromasters application.
+### Load grades
 
-### Load adjusted grades
+Note that users will have usernames from the platform on which they took the exams (e.g. MITx Online). If necessary, use the social_auths to convert the usernames of the exam platform to Micromasters usernames. 
+
+Steps:
```
Comment (annagav) [inline grades/README.md:156]: We have these instructions written already in exams/README.md

**Review by annagav — CHANGES_REQUESTED** [review]
There is a part of the document that is duplicated.

File: grades/README.md (hunk 2 above)
Comment (briangrossman) [reply grades/README.md:156]: @annagav  Thanks. I've added a reference to (exams/README.md), but I've kept the instructions in place here for completeness.

**Review by annagav — APPROVED** [review]
👍

**Pull request merged**